README.md

```diff
@@ -6,11 +6,12 @@ svgpath
 [![Coverage Status](https://img.shields.io/coveralls/fontello/svgpath/master.svg?style=flat)](https://coveralls.io/r/fontello/svgpath?branch=master)
 
 
-> Low level toolkit for SVG paths transformations. Sometime you can't use
-`transform` attributes and have to apply changes to svg paths directly.
+Low level toolkit for SVG paths transformations.
```

Comment:
Oh, yes, I forgot to mention that. With the `>`, that description becomes greyed out and it's a little hard to read (especially for what's supposed to be the main description of the repo). Perhaps some different formatting could be used, like italics?